examples/hello-world/step-by-step/cifar10/swarm/swarm.ipynb

```diff
@@ -41,90 +41,90 @@
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "369c5501",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "! cat /tmp/nvflare/jobs/swarm_cse_pt/app/config/config_fed_server.conf"
+    "## Prepare Data\n",
+    "\n",
+    "Make sure the CIFAR10 dataset is downloaded with the following script:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "d223847b",
+   "id": "c4861f23",
    "metadata": {},
    "outputs": [],
    "source": [
-    "! cat /tmp/nvflare/jobs/swarm_cse_pt/app/config/config_fed_client.conf"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "83cc8869",
-   "metadata": {},
-   "source": [
-    "## Prepare Data"
+    "! python ../data/download.py"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "8f63bf0f",
+   "id": "b2e15709",
    "metadata": {},
    "source": [
-    "Make sure the CIFAR10 dataset is downloaded with the following script:"
+    "## Job API\n",
+    "\n",
+    "Let's use the Job API to create a CCWF Job.\n",
+    "\n",
+    "We use the `add_swarm()` function to add our server_config, client_config, and cse_config.\n",
+    "\n",
+    "First add the `SwarmServerConfig` with the number of rounds for the `SwarmServerController`.\n",
+    "\n",
+    "We also add `CrossSiteEvalConfig` for the `CrossSiteEvalServerController`.\n",
+    "\n",
+    "On the client side, we add the `SwarmClientConfig` for the `SwarmClientController` which maps the `learn_task_name` to `train` and add the `CrossSiteEvalClientController` which uses the `validate` and `submit_model` tasks. These task are handled by the `ScriptRunner` with our `train.py` script. Additionally, required components including the persistor with the initial `Net()` model, aggregator, and shareable generator are defined as client-side components.\n",
+    "\n",
+    "Below is our Job API script:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "17323f61",
+   "id": "97a3aafd",
    "metadata": {},
    "outputs": [],
    "source": [
-    "! python ../data/download.py"
+    "import sys\n",
+    "\n",
+    "code_path = \"../code/fl\"\n",
+    "if code_path not in sys.path:\n",
+    "    sys.path.append(code_path)\n",
+    "\n",
+    "from net import Net\n",
+    "\n",
+    "from nvflare.apis.dxo import DataKind\n",
+    "from nvflare.app_common.aggregators.intime_accumulate_model_aggregator import InTimeAccumulateWeightedAggregator\n",
+    "from nvflare.app_common.ccwf.ccwf_job import CCWFJob, CrossSiteEvalConfig, SwarmClientConfig, SwarmServerConfig\n",
+    "from nvflare.app_common.ccwf.comps.simple_model_shareable_generator import SimpleModelShareableGenerator\n",
+    "from nvflare.app_opt.pt.file_model_persistor import PTFileModelPersistor\n",
+    "from nvflare.job_config.script_runner import ScriptRunner\n",
+    "\n",
+    "n_clients = 2\n",
+    "num_rounds = 3\n",
+    "train_script = \"../code/fl/train.py\"\n",
```

Comment:
this will not work anymore as you pointed out

Reply:
thanks, fixed to copy files over